factory.js

```diff
@@ -113,41 +114,10 @@ Factory.prototype.init = function init(next) {
   // Read the package.json AND the wrhs.toml
   //
   debug(`Read package.json: ${ pkg } and wrhs.toml: ${ wrhs }`);
```

Comment:
probably don't need that or the `pkg` & `wrhs` variables here anymore.

maybe keep the debug line but have it just say `Extracting warehouse configuration`